src/snowflake/snowpark/dataframe.py

```diff
@@ -2929,6 +2930,58 @@ def create_or_replace_view(
             ),
         )
 
+    @df_collect_api_telemetry
+    def create_or_replace_dynamic_table(
+        self,
+        name: Union[str, Iterable[str]],
+        *,
+        warehouse: str,
+        lag: str,
+        statement_params: Optional[Dict[str, str]] = None,
+    ) -> List[Row]:
+        """Creates a dynamic table that captures the computation expressed by this DataFrame.
```

Comment:
I suggest adding a link to the doc of dynamic table

Reply:
The docs are currently in limited access for private preview.  We can update the comments with the correct docs when we go PuPr